### DISCUSSION TITLE
About the whitelist function

### DISCUSSION THREAD

**Discussion by gityms123** [strongswan/strongswan#1153] [Q&A]
I want to implement a function, that is, the client can only access the specified ip after successfully connecting to the vpn. I tried to use ufw default allow outgoing to prohibit all outbound rules and only open it to the specified IP, but the vpn does not seem to be affected, and the access can be completed normally. All addresses.
Is there any good way?

**Comment by strongX509** [comment]
Have a look at the strongSwan updown script which automatically inserts a passthrough firewall rule when the IPsec tunnel has been established successfully.

https://docs.strongswan.org/docs/5.9/plugins/updown.html

**Comment by Thermi** [comment]
You can use the policy match to check if there is no IPsec policy for that packet and do it that way.
